staging-service/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1 @@
+org.springframework.context.ApplicationListener=com.github.cafdataprocessing.services.staging.StartupListener
```

Comment:
Shouldn't need this, the StartupListener is a Component and will be pickup up by the package scan

Reply:
Removed